modules/renter.go

```diff
@@ -56,46 +75,64 @@ type FileInfo struct {
 	Expiration     types.BlockHeight `json:"expiration"`
 }
 
-// DownloadInfo provides information about a file that has been requested for
-// download.
-type DownloadInfo struct {
-	SiaPath     string    `json:"siapath"`
-	Destination string    `json:"destination"`
-	Filesize    uint64    `json:"filesize"`
-	Received    uint64    `json:"received"`
-	StartTime   time.Time `json:"starttime"`
-}
-
-// An Allowance dictates how much the Renter is allowed to spend in a given
-// period. Note that funds are spent on both storage and bandwidth.
-type Allowance struct {
-	Funds       types.Currency    `json:"funds"`
-	Hosts       uint64            `json:"hosts"`
-	Period      types.BlockHeight `json:"period"`
-	RenewWindow types.BlockHeight `json:"renewwindow"`
-}
-
-// RenterSettings control the behavior of the Renter.
-type RenterSettings struct {
-	Allowance Allowance `json:"allowance"`
-}
-
 // A HostDBEntry represents one host entry in the Renter's host DB. It
 // aggregates the host's external settings and metrics with its public key.
+//
+// TODO: WE FORGOT THE JSON STRINGS FOR SOME OF THE ELEMENTS! It's used in both
+// persist and over the API.
```

Comment:
I actually don't think this breaks compatibility: https://play.golang.org/p/R5hrFPYjyW